src/uploadDsym.groovy

```diff
@@ -0,0 +1,99 @@
+import groovyx.net.http.HTTPBuilder
+import net.sf.json.JSON
+import org.apache.http.entity.mime.HttpMultipartMode
+import org.apache.http.entity.mime.MultipartEntityBuilder
+import org.apache.http.entity.mime.content.FileBody
+import org.apache.http.entity.mime.content.StringBody
+import static groovyx.net.http.Method.POST
+import com.urbancode.air.AirPluginTool;
+
+final def workDir = new File('.').canonicalFile;
+final def props = new Properties();
+final def inputPropsFile = new File(args[0]);
+try {
+    inputPropsStream = new FileInputStream(inputPropsFile);
+    props.load(inputPropsStream);
+}
```

Comment:
For property resolution, we'll want to use AirPluginTool. You can replace lines 11 to 19 with lines 201 and 22 in this file: https://github.com/IBM-UrbanCode/Airwatch-UCD/blob/master/src/main/groovy/airwatch-ucd.groovy

There are updates coming down the line in AirPluginTool that we want to be prepared for.